tests/SIL.Machine.WebApi.SpecFlowTests/StepDefinitions/MachineAPIStepDefinitions.cs

```diff
@@ -18,22 +18,6 @@ public sealed class MachineApiStepDefinitions
         public MachineApiStepDefinitions()
         {
             this.client = new WebApiClient(MACHINE_API_TEST_URL, true);
-            SetAccessTokenFromEnvironment();
```

Comment:
We should keep this code but make it a warning instead of an error.   We may also use this to test a QA environment and want to verify that the authentication is working.

Reply:
@johnml1135 done. It is not clear to me the best way to log a warning in C# also, it looks like the ternary operator needs to throw an exception in order to work. Thus, I've gone with the try...catch approach. LMK if it's not what you hoped for.